scripts/convert.sh

```diff
@@ -73,6 +75,11 @@ do
         if [ "$char" = $'r' ] || [ "$char" = $'n' ]; then
             printf "\\\\$char" >> $OUTPUT
         fi
+    elif [ "$char" = $'\n' ] && [ $UGLIFY -eq 0 ]; then
+        # Handle new lines if uglify is not installed
+        printf "\"" >> $OUTPUT
+        printf "\n" >> $OUTPUT
+        printf "\"" >> $OUTPUT
```

Comment:
Wouldn't printf "\"\n\"" >> $OUTPUT be the same as these three lines?

And what is this doing? I'm confused. :D